nss-day-cohort-60/django-taters-api
# author URL changed to users author_view updated

Closes #78


# Description of Proposed Changes

changed router path from 'authors' to 'users' to conform to client side requests.
updated author_view.py to return data from the USER model in addition to the author model to conform to client side requests.
---


# Steps to Test

1. Fetch the jes-refactor-users branch and switch to it
2. in postman perform a GET on http://localhost:8088/users and verify all information is there
3. do the same with http://localhost:8088/users/1
